src/main/webapp/WEB-INF/web.xml

```diff
@@ -14,6 +14,9 @@
             <role-name>*</role-name>
         </auth-constraint>
     </security-constraint>
+    <welcome-file-list>
```

Comment:
nit: move `welcome-file-list` before of after `security-constraint`, right now it is in the middle of security-constraints.